federation/deploy/deploy.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script turns up/turns down Kubernetes clusters and federation
+# components using the built hyperkube image.
+# e.g. run as:
+# FEDERATION_OUTPUT_ROOT="./_output" ./deploy.sh deploy_clusters
+#
+# will deploy the kubernetes clusters using the configuration specified
+# in $FEDERATION_OUTPUT_ROOT/config.json.
+#
+# See config.json.sample for a config.json example.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
+
+# Provides the $KUBERNETES_PROVIDER variable and detect-project function
+source "${KUBE_ROOT}/cluster/kube-util.sh"
+# Provides logging facilities
+source "${KUBE_ROOT}/cluster/lib/logging.sh"
+
+readonly KUBE_ANYWHERE_FEDERATION_IMAGE="gcr.io/madhusudancs-containers/kubernetes-anywhere-federation"
```

Comment:
Can you add a comment regarding where does the code for this image exist?
Also, why cant we use the vanila kubernetes-anywhere image?